README.markdown

```diff
@@ -4535,20 +4547,23 @@ Here the `args` table will always look like
 
 regardless of the actual request query string.
 
-Note that a maximum of 100 request arguments are parsed by default (including those with the same name) and that additional request arguments are silently discarded to guard against potential denial of service attacks.
+Note that a maximum of 100 request arguments are parsed by default (including those with the same name) and that additional request arguments are silently discarded to guard against potential denial of service attacks. Since `v0.10.13`, when the limit is exceeded, it will return a second value which is the string `"truncated"`.
 
 However, the optional `max_args` function argument can be used to override this limit:
 
 ```lua
 
- local args = ngx.req.get_uri_args(10)
+ local args, err = ngx.req.get_uri_args(10)
+ if err == "truncated" then
+     -- one can choose to ignore or reject the current request here
+ end
 ```
 
 This argument can be set to zero to remove the limit and to process all request arguments received:
 
 ```lua
 
- local args = ngx.req.get_uri_args(0)
+ local args, err = ngx.req.get_uri_args(0)
```

Comment:
since `err` would never be set here (at least in the current context), is it appropriate to leave this example as-is?

Reply:
@p0pr0ck5 That's for forward compatibility (we may add more error strings in the future).

Reply:
Yes but the context here is specifically about a unlimited number of args returned.

Reply:
@p0pr0ck5 It's better to always be prepared for an error string to be returned in our docs. It's  much better than assuming only one value can be returned.